### PR TITLE
Fix Plotly.animate on graphs with multiple basePlotModules

### PR DESCRIPTION
**Pull request by etpinard**
fixes https://github.com/plotly/plotly.js/issues/3849

before: https://codepen.io/MojtabaSamimi/pen/ardKoa?editors=0010
after: https://codepen.io/etpinard/pen/EzgzGQ?editors=1010

### CODE REVIEW

**Comment by etpinard** [comment]
cc @plotly/plotly_js

File: test/image/mocks/sunburst_pie_cartesian.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
+{
```
Comment (archmoj) [inline test/image/mocks/sunburst_pie_cartesian.json]: Nicely done. 
Could we reuse this mock instead?
test/image/baselines/display-text_zero-number.png

Comment (etpinard) [reply test/image/mocks/sunburst_pie_cartesian.json]: What's wrong with the mock I used?

Comment (etpinard) [reply test/image/mocks/sunburst_pie_cartesian.json]: Ha I see, that mock also has a sunburst along with cartesian traces. Yep, good call. Thanks!

Comment (etpinard) [reply test/image/mocks/sunburst_pie_cartesian.json]: Done in https://github.com/plotly/plotly.js/pull/3860/commits/d5064ff3e7da3b036b66e4def205e558d36367cf

**Comment by archmoj** [comment]
:dancer:

**Pull request merged**